sdk/schemaregistry/client/src/main/java/com/azure/schemaregistry/client/rest/entities/responses/RegisterSchemaResponse.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.azure.schemaregistry.client.rest.entities.responses;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+
+public class RegisterSchemaResponse {
```

Comment:
Do you have a service swagger that can be used to generate clients using autorest?

Reply:
@hmlam